bin/p2-preparer/main.go

```diff
@@ -27,12 +27,12 @@ func main() {
 	}
 	preparerConfig, err := preparer.LoadConfig(configPath)
 	if err != nil {
-		logger.WithField("inner_err", err).Fatalln("could not load preparer config")
+		logger.WithField("err", err).Fatalln("could not load preparer config")
 	}
 
 	prep, err := preparer.New(preparerConfig, logger)
 	if err != nil {
-		logger.WithField("inner_err", err).Fatalln("Could not initialize preparer")
+		logger.WithField("err", err).Fatalln("Could not initialize preparer")
```

Comment:
We should make a first-class error method to initialize a field set (and get rid of the magic `"err"`s everywhere).

```
logger.WithError(err).Fatalln("Could not initialize preparer")
```